backend/sources/facebook/sender/BUILD

```diff
@@ -3,12 +3,10 @@ load("//tools/build:junit5.bzl", "junit5")
 load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 app_deps = [
-    "//backend/avro/communication:channel",
-    "//backend/avro/communication:message",
-    "//backend/lib/payload",
-    "//backend/lib/kafka/schema:kafka-schema",
-    "//backend/lib/kafka/schema:application-communication-channels",
-    "//backend/lib/kafka/schema:application-communication-messages",
+    "//backend:base_app",
+    "//backend:channel",
```

Comment:
Reads much nicer. The domain concern is composed of both so instead of bundling one into the other we create this shorthand 👍